examples/execute/execute.md

```diff
@@ -0,0 +1,28 @@
+# Execute function
```

Comment:
Letsadd symlinks to the latest example mds. This will show this file when the folder is viewed

Reply:
@aashikam  We can use `ln -s ../execute.md .github/README.md` to create a symbolic link between [execute.md](https://github.com/aashikam/module-ballerinax-aws.redshift/blob/docs/examples/execute/execute.md) and .github/README.md so it will appear as the readme page for the example in GitHub. 

eg:- https://github.com/ballerina-platform/module-ballerinax-twilio/tree/master/examples/accounts/create-sub-account

Reply:
08c5157e77599830cb652a3eace45ad82c4c028a